lib/api/bucketDeleteCors.js

```diff
@@ -0,0 +1,52 @@
+import { errors } from 'arsenal';
+
+import bucketShield from './apiUtils/bucket/bucketShield';
+import { isBucketAuthorized } from './apiUtils/authorization/aclChecks';
+import metadata from '../metadata/wrapper';
+// import { pushMetric } from '../utapi/utilities';
```

Comment:
The code is commented out because I will need to update Utapi and Integration both to make this function call, and there's several cors api that will need similar pull requests. It's easier to merge this first and then make joint prs to address for all cors api in S3 (+ Integration & Utapi). So it's commented out for now, but is intended to be uncommented later. :)